db/db_basic_test.cc

```diff
@@ -1701,8 +1702,8 @@ TEST_F(DBBasicTest, MultiGetIOBufferOverrun) {
   BlockBasedTableOptions table_options;
   table_options.pin_l0_filter_and_index_blocks_in_cache = true;
   table_options.block_size = 16 * 1024;
-  assert(table_options.block_size >
-          BlockBasedTable::kMultiGetReadStackBufSize);
+  ASSERT_GT(table_options.block_size,
```

Comment:
I think this change caused the failure. I am not sure why, but you can revert this change.

Reply:
Yes I found out it too. I'm going to use ASSERT_TRUE() instead.

Reply:
I wonder whether that will work since the following does not
```
size_t tmp = BlockBasedTable::kMultiGetReadStackBufSize;
(void)tmp;
```

Reply:
Please disregard my previous comment (deleted)